### DISCUSSION TITLE
I didn't find a way how to recognize tap on the map using this package.

### DISCUSSION THREAD

**Discussion by miakh** [josxha/flutter-maplibre#182] [Q&A] [answered]
1. It is possible to detect MapEventClick. But in Flutter that is basically onPointerDown, not onTap.
2. If I put GestureDetector above Map, then again onTap is not detected.

Also another issue is that MapEventDoubleClick is being only detected when it is done by mouse, not by touch.

Anyway great work with the package!

**Comment by josxha** [comment, marked as answer]
Thanks for your feedback @miakh. I agree, gesture callbacks is something the package needs to improve on. A GestureDetector won't work as it won't pass the gestures on to the child widgets but you can use a Listener instead. 

Maybe that's in general a suitable solution to fix platform specific gesture callbacks as it currently is. 🤔

**Comment by miakh** [comment]
That actually works fine, thanks.
